<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppItem</class>
 <widget class="QWidget" name="AppItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>启用</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_app_name">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;隧道名称(离线)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_uuid">
       <property name="text">
        <string>uuid</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_dstport">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>远程端口: 11451</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_srcport">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>本地端口: 11451</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_protocol">
         <property name="text">
          <string>协议: tcp</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_del">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_copy_addr">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>复制</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/image/copy.png</normaloff>:/image/copy.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_edit">
         <property name="minimumSize">
          <size>
           <width>4</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
